datadog/fwprovider/resource_datadog_on_call_schedule.go

```diff
@@ -151,17 +151,17 @@ func (r *onCallScheduleResource) Schema(_ context.Context, _ resource.SchemaRequ
 							NestedObject: schema.NestedBlockObject{
```

Comment:
NOTE TO REVIEWER: this was a temp fix change I had to make before the official fix goes in - this won't be merged in so you can ignore this!!!